build.gradle

```diff
@@ -1,46 +1,123 @@
+import org.grails.gradle.plugin.doc.PublishGuideTask
+
+buildscript {
+    repositories {
+        maven { url "https://repo.grails.org/grails/core" }
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.grails.grails-doc:org.grails.grails-doc.gradle.plugin:$grailsVersion"
+        classpath "org.grails:grails-gradle-plugin:$grailsVersion"
+    }
+}
+
 plugins {
-    id 'groovy'
-    id 'java-library'
-    id 'io.github.gradle-nexus.publish-plugin'
-    id 'maven-publish'
-    id 'signing'
+    id "java-library"
+    id "eclipse"
+    id "idea"
 }
 
+apply plugin: "org.grails.grails-plugin"
+apply plugin: 'org.grails.grails-doc'
+apply plugin: 'org.grails.grails-publish'
```

Comment:
There's an issue upstream with this plugin right now where this should be applied after the version definition if possible.  I typically move it just before the grailsPublish block.

Reply:
I swapped where the version info gets set, however I like having all the plugins applied in one section so it easier to read later.  I am happy to move it if the community prefers it near the block so it easier to connect the plugin with the configuration settings.

Reply:
That works too